### PR TITLE
Navigation enhancements

### PR DESCRIPTION
**Pull request by cecilphillip**
- Properly applies active class to navigation menu
- Added error message that shows when trying to create links before configuring settings
- Retrain input field state in Home view when navigating away

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -46,7 +46,6 @@
             v-on:change="setShortenerProvider"
             v-model="shortenerProvider"
           >
-            <option>none</option>
```
Comment (spboyer) [inline src/views/Settings.vue]: Should we not have the option to have no shortener configured?

**Comment by spboyer** [comment]
Looks good other than one comment about removing the `none` option for link shortener provider.

File: src/views/Settings.vue (hunk 1 above)
Comment (cecilphillip) [reply src/views/Settings.vue]: If that's the case, we could just use the quick links on home right?

**Comment by spboyer** [comment]
merged and pushed to live. Thanks!

**Pull request merged**